apps/api/pipelines/azure-pipelines-release.yml

```diff
@@ -10,6 +10,10 @@ parameters:
       - UK West
       - UK South
     default: UK West
+  - name: schemaMigration
+    displayName: Run Schema migration?
```

Comment:
Is the question mark because you are not sure what the best name for this step is? I would call is something like "Run Database Migration" if pressed, but this is fine also

Reply:
Done